test/page.spec.js

```diff
@@ -104,19 +104,16 @@ describe('Async stacks', () => {
   });
 });
 
-describe('Page.Events.error', function() {
+describe('Page.Events.Crash', function() {
```

Comment:
Let's add a test that you cannot do anything with the page after crash, and ensure that.